_notebooks/textbook.py

```diff
@@ -130,3 +144,22 @@ def clean_raw_text(self, disable_pysbd=False):
             if not disable_pysbd:
                 clean_text = "\n".join(seg.segment(clean_text))
             chapter.clean_text = clean_text
+    
+    def resolve_coreference(self):
+        """Uses spacy pipleline as a base and extends it
+        using neuralcoreference to process the coreferences
+        in the plain text that you get.
+
+        Saving both resolved text as well as the coreference
+        clusters in the chapter.
+        """
+        for chapter in tqdm(self.chapters):
+            if not chapter.clean_text :
+                print(
+                    "There is no clean_text for the chapter. \
```

Comment:
Can you confirm if this is a possibly inconsistent workflow?

If you are allowing me to NOT run `clean_raw_text` -- as mentioned in the explanation of `disable` flag, I should be able to run this code without an exception, and just a warning. 

Option 2) Remove the disable flag and hard code that we always run the sentence boundary detection. 

I'd prefer being able to control this via a flag. 
Rewrite the code flow such that I can the flag only once and the rest is taken care of.

Reply:
Disable flag is there if we want to do additional cleaning for just certain text. Like we know certain chapters have a fixed word that we want to remove, so we will write a custom cleaning block and add it to the chapter.clean_text (or now space_formatted_text) and then continue working without making any changes in textbook.py